docs/rfcs/001-recent-commits.md

```diff
@@ -29,11 +29,11 @@ If the active repository has no commits yet, display a short panel with a backgr
 
 Otherwise, display a **recent commits** section containing a sequence of horizontal bars for ten **relevant** commits with the most recently created commit on top. The commits that are considered **relevant** include:
 
-* Commits reachable by the remote tracking branch that is the current upstream of `HEAD`. If more than three of these commits are not reachable by `HEAD`, they will be hidden behind an expandable accordion divider.
+* Commits reachable by the remote tracking branch that is the current upstream of `HEAD`.
 * Commits reachable by `HEAD` that are not reachable by any local ref in the git repository.
 * The single commit at the tip of the branch that was branched from.
 
-The most recent three commits are visible by default and the user can scroll to see up to the most recent ten commits. The user can also drag a handle to resize the recent commits section and show more of the available ten.
+The most recent three commits are visible by default and the user can scroll to see up to the most recent ten commits.
```

Comment:
I think we do still want to make this resizable. Although I will say I find myself not missing it as much as I thought I would.